transfers/transfer.py

```diff
@@ -491,41 +541,76 @@ def main(am_user, am_api_key, ss_user, ss_api_key, ts_uuid, ts_path, depth, am_u
+                        required=True, help='API key of the Storage Service \
+                        user.')
+    parser.add_argument(
+        '-t', '--transfer-source', metavar='UUID', required=True,
+        help='Transfer Source Location UUID to fetch transfers from.')
+    parser.add_argument(
+        '--transfer-path', metavar='PATH', help='Relative path within the \
+        Transfer Source. Default: ""', type=fsencode, default=b'')  \
+        # Convert to bytes from unicode str provided by command line
+    parser.add_argument(
+        '--depth', '-d', help='Depth to create the transfers from relative to \
+        the transfer source location and path. Default of 1 creates transfers \
+        from the children of transfer-path.', type=int, default=1)
+    parser.add_argument('--am-url', '-a', metavar='URL',
+                        help='Archivematica URL. Default: %s' %
+                        defaults.DEF_AM_URL,
+                        default='%s' % defaults.DEF_AM_URL)
+    parser.add_argument('--ss-url', '-s', metavar='URL',
+                        help='Storage Service URL. Default: %s' %
+                        defaults.DEF_SS_URL,
+                        default='%s' % defaults.DEF_SS_URL)
+    parser.add_argument(
+        '--transfer-type', metavar='TYPE', help="Type of transfer to start. \
+        One of: 'standard' (default), 'unzipped bag', 'zipped bag', 'dspace'.",
+        default='standard', choices=['standard', 'unzipped bag', 'zipped bag',
+                                     'dspace'])
+    parser.add_argument('--files', action='store_true',
+                        help='If set, start transfers from files as well as \
+                        folders.')
+    parser.add_argument('--hide', action='store_true',
+                        help='If set, hide the Transfers and SIPs in the \
+                        dashboard once they complete.')
+    parser.add_argument('-c', '--config-file', metavar='FILE',
+                        help='Configuration file(log/db/PID files)',
+                        default=None)
 
     # Logging
-    parser.add_argument('--verbose', '-v', action='count', default=0, help='Increase the debugging output.')
-    parser.add_argument('--quiet', '-q', action='count', default=0, help='Decrease the debugging output')
-    parser.add_argument('--log-level', choices=['ERROR', 'WARNING', 'INFO', 'DEBUG'], default=None, help='Set the debugging output level. This will override -q and -v')
+    parser.add_argument('--verbose', '-v', action='count',
+                        default=0, help='Increase the debugging output.')
+    parser.add_argument('--quiet', '-q', action='count',
+                        default=0, help='Decrease the debugging output')
+    parser.add_argument(
+        '--log-level', choices=['ERROR', 'WARNING', 'INFO', 'DEBUG'],
+        default=None, help='Set the debugging output level. This will \
+        override -q and -v')
```

Comment:
MegaNitpick: "The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation." From https://www.python.org/dev/peps/pep-0008/.

In the case of breaking strings across lines, I think the parenthesis with implicit concatenation strategy is better because you don't end up having strange sequences of whitespace in the middle of your strings. In this context it doesn't matter because argparse is getting rid of them for you, but in general, it's one reason beyond an appeal to authority for this preference.

Reply:
I do agree. I should have fixed that now with the next commit when it arrives.